numerics/apodization.hpp

```diff
@@ -15,9 +18,56 @@ template<template<typename, typename, int> class Evaluator>
 PoissonSeries<double, 0, Evaluator> Dirichlet(Instant const& t_min,
                                               Instant const& t_max);
 
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Sine(Instant const& t_min,
+                                         Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Hahn(Instant const& t_min,
+                                         Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Hamming(Instant const& t_min,
+                                            Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Blackman(Instant const& t_min,
+                                             Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> ExactBlackman(Instant const& t_min,
+                                                  Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Nuttall(Instant const& t_min,
+                                            Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> BlackmanNuttall(Instant const& t_min,
+                                                    Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> BlackmanHarris(Instant const& t_min,
+                                                   Instant const& t_max);
+
+// The flat-top window in Wikipedia is not normalized and comes from Matlab (?).
+// We use the normalized ISO 18431-2 instead, which is very close.
```

Comment:
[ISO 18431-2:2004](https://www.iso.org/fr/standard/35589.html) specifies three windows; disambiguate (I think this is the one specified by section 5.3, see https://www.sis.se/api/document/preview/905378/).

While you are at it, refer to the Hanning and Rectangular sections of this standard in the comments for the appropriate functions.